src/components/ens-registration/RegistrationCover/RegistrationCover.tsx

```diff
@@ -102,6 +107,8 @@ const RegistrationCover = ({
     },
     [onBlurField, setCoverMetadata]
   );
+  const [isUploading, setIsUploading] = useState(false);
+  const [isLoadingImage, setIsLoadingImage] = useState(false);
```

Comment:
i'm not following why we need two states `isUploading` and `isLoadingImage` if they're both being used to show `Uploading` on the ui https://github.com/rainbow-me/rainbow/pull/3963/files#diff-28a65281fd94c155c98670d0b2c1602d01b2fe3a7509da60294d854aee700bfeR189

Reply:
names make sense but wondering if we could just use one `isImageReady` or something like that

Reply:
do we even need this state? `useSelectImageMenu` also returns `isUploading`.